webdriver/src/com/googlecode/jmeter/plugins/webdriver/sampler/gui/WebDriverSamplerGui.java

```diff
@@ -139,15 +134,20 @@ public void actionPerformed(ActionEvent actionEvent) {
 
     private void setScriptContentType(String ctype) {
         String text = script.getText();
-        script.setContentType(ctype);
+        script.setSyntaxEditingStyle(ctype);
         script.setText(text);
     }
 
     private JPanel createScriptPanel() {
-        script = new JEditorPane();
-        final JScrollPane scrollPane = new JScrollPane(script);
-        setScriptContentType("text/plain");
-        script.setFont(new Font(Font.MONOSPACED, Font.PLAIN, 14));
+    	script = new JSyntaxTextArea(20, 20, false);
+    	script.getFoldManager().setCodeFoldingEnabled(true);
+    	script.setFont(new Font("Hack", Font.PLAIN, 14));
```

Comment:
I'm sorry - my fault I'm going to add some checks of course. For example, if a platform has Hack font prefer it  - otherwise use any available monospace font. FYI  https://github.com/chrissimpkins/Hack.
Is it okay? or I should find another way to set preferred font for text area? May I use there JMeter's properties?